defaults/main.yml

```diff
@@ -0,0 +1,333 @@
+---
+###############################################################################
+# TODO: Add the default values in here for any of the converted heat params
```

Comment:
Is this TODO done now, or are there other heat params that need to be added?

Reply:
This has been removed